atomicapp/providers/openshift.py

```diff
@@ -153,8 +170,7 @@ class OpenShiftProvider(Provider):
     def init(self):
         self.openshift_artifacts = {}
 
-        (self.providerapi, self.access_token, self.namespace) = \
-            self._get_config_values()
+        self._set_config_values()
```

Comment:
I tried to add fix here (fd05e482dc7305123bf0413aa259771a7306d412)
But haven't made much tests yet.